mmdet/datasets/dsdl.py

```diff
@@ -0,0 +1,192 @@
+# Copyright (c) OpenMMLab. All rights reserved.
+import os
+from typing import List
+
+try:
+    from dsdl.dataset import DSDLDataset
+except ImportError:
+    DSDLDataset = None
+
+from mmdet.registry import DATASETS
+from .base_det_dataset import BaseDetDataset
+
+if DSDLDataset is None:
+    raise RuntimeError(
+        'Package dsdl is not installed. Please run "pip install dsdl".')
```

Comment:
Place these lines into the DSDLDetDataset class. Therefore, the error will only occur when the user uses this class. Users who do not require this feature are not affected.

Reply:
Please refer to https://github.com/open-mmlab/mmdetection/blob/92d03df639a099c7263805a859923e2f0815e381/mmdet/evaluation/metrics/lvis_metric.py#L80-L84